test/opentype.spec.mjs

```diff
@@ -15,33 +15,6 @@ describe('opentype.js', function() {
         assert.equal(aGlyph.path.commands.length, 15);
     });
 
-    it('[deprecated] .load() promise resolve uppon success', function(done) {
```

Comment:
I thought we agreed that this would still function in 2.0.0 with a warning, and be removed in a subsequent release.